.eslintrc.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   root: true,
-  extends: "babel",
+  parser: "lightscript-eslint",
```

Comment:
Don't look at me, I just wanted the unit tests to run out of the box so I knew if my changes were breaking anything...